### DISCUSSION TITLE
ESPHome basierter onBattery Monitor

### DISCUSSION THREAD

**Discussion by swingstate** [hoylabs/OpenDTU-OnBattery#538] [Show and tell]
Ich habe meinen 3D Drucker bemüht und ein bisschen in Homeassistant "programmiert". 

Dabei raus gekommen ist ein 2,1" Display welches die Daten, welche onBattery via MQTT an HA liefert anzeigt. 

![IMG_2744](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/73672554/e4744473-e544-4303-9245-550afbcef039)

In meinem Fall den SoC, die Batterietemperatur, die Solarleistung und die Einspeisung durch den Inverter. 

Basis ist das WaveShare Cloudmodule (ePaper Display mit einem ESP32), für dass ich ein Case und eine Dockingstation entworfen habe. In dem Case steckt auch der 3.7V 2000mAh LiPo Akku, der im Lieferumfang des Moduls ist. Damit läuft das Display ca 30h autark, wenn es mal mit auf die Terrasse soll. 

![IMG_2745](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/73672554/abcf79bb-cce7-4c42-933a-73734ef77f76)

Geladen wird via USB über die Dockingstation. In der Dockingstation steckt ein USB Winkel Adapter, so dass sich USB-C Kabel jeder Länge anschliessen lassen. 

Am Case/Dock und dem Code für das Display gibt es noch Optimierungspotenzial und wenn's soweit fertig ist erstelle ich dafür ein Github Repo incl. Code und den files für Case & Dock.

**Comment by Tiese** [comment]
Das ist ja eine tolle Idee 👍
